docs/DeveloperGuide.adoc

```diff
@@ -11,11 +11,7 @@ endif::[]
 ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 :repoURL: https://github.com/se-edu/addressbook-level4/tree/master
 
-<<<<<<< HEAD
 By: `Team 2`      Since: `Jun 2017`      Licence: `MIT`
-=======
-By: `Team 2`      Since: `Jun 2016`      Licence: `MIT`
->>>>>>> V0.0
```

Comment:
Not sure when was this introduced, I just resolved it by following what's seen on V0.0_mattheus branch.